tigerbeetle.yaml

```diff
@@ -37,3 +37,10 @@ update:
     use-tag: true
     # There are some prereleases with a weird WEEKLY format.
     tag-filter: 0.
+
+test:
+  pipeline:
+    - name: Check if TigerBeetle was installed
+      runs: |
+        tigerbeetle --help
+        
```

Comment:
There's some whitespace on this line that needs to be deleted so that the linter passes!